src/main/java/com/conveyal/r5/analyst/broker/Broker.java

```diff
@@ -380,8 +379,27 @@ public void createWorkersInCategory (WorkerCategory category) {
             throw new RuntimeException(e);
         }
 
+        String workerConfigString = cfg.toString();
+        String scriptTemplate;
+
+        // Read in the startup script
+        try {
+            InputStream scriptIs = Broker.class.getClassLoader().getResourceAsStream("worker.sh");
+            ByteArrayOutputStream scriptBaos = new ByteArrayOutputStream();
+            ByteStreams.copy(scriptIs, scriptBaos);
+            scriptIs.close();
+            scriptBaos.close();
+            scriptTemplate = scriptBaos.toString();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+
+        String logGroup = workerConfig.getProperty("log-group");
+
+        String script = MessageFormat.format(scriptTemplate, workerDownloadUrl, logGroup, workerConfigString);
```

Comment:
This MessageFormat seems to be intended for internationalized user messages, and the copying of streams into byte arrays etc. is a bit strange, but maybe there's no better way to do this. Positionally filling in the fields is a little brittle, but there are only four of them so I guess it'll do the job.